tests/test_cmt.py

```diff
@@ -1,8 +1,89 @@
 """Testing functions for the CMT services"""
 
+import strax
 import straxen
+import utilix
 import numpy as np
+from warnings import warn
+from .test_basics import test_run_id_1T
+from .test_plugins import test_run_id_nT
+from straxen.common import aux_repo
 
+def test_connect_to_db():
+    """
+    Test connection to db
+    """
+    if not straxen.utilix_is_configured():
+        warn('Cannot do test becaus  '
+             'no have access to the database.')
+        return

```

Comment:
This is only the case if you push from one branch of the origin repository to another. Hence when pushing from your fork those tests wont be executed.